docs/running-on-kubernetes.md

```diff
@@ -112,16 +108,36 @@ If our local proxy were listening on port 8001, we would have our submission loo
       --kubernetes-namespace default \
       --conf spark.executor.instances=5 \
       --conf spark.app.name=spark-pi \
-      --conf spark.kubernetes.driver.docker.image=registry-host:5000/spark-driver:latest \
-      --conf spark.kubernetes.executor.docker.image=registry-host:5000/spark-executor:latest \
+      --conf spark.kubernetes.driver.docker.image=kubespark/spark-driver:0.1.0-alpha.1 \
+      --conf spark.kubernetes.executor.docker.image=kubespark/spark-executor:0.1.0-alpha.1 \
       examples/jars/spark_examples_2.11-2.2.0.jar
 
 Communication between Spark and Kubernetes clusters is performed using the fabric8 kubernetes-client library.
 The above mechanism using `kubectl proxy` can be used when we have authentication providers that the fabric8
-kubernetes-client library does not support. Authentication using X509 Client Certs and oauth tokens
+kubernetes-client library does not support. Authentication using X509 Client Certs and OAuth tokens
 is currently supported.
 
-### Determining the Driver Base URI
+## Advanced
+ 
+### Setting Up SSL For Submitting the Driver
+
+When submitting to Kubernetes, a pod is started for the driver, and the pod starts an HTTP server. This HTTP server
```

Comment:
I think [the former](https://books.google.com/ngrams/graph?content=an+http%2Ca+http&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Can%20http%3B%2Cc0%3B.t1%3B%2Ca%20http%3B%2Cc0#t1%3B%2Can%20http%3B%2Cc1%3B.t1%3B%2Ca%20http%3B%2Cc1) is more common.